annarosepaloma/javascript30
# 30 - Whack A Mole Game

# Related Tickets
https://codeandtheory.atlassian.net/browse/JS-1

# Description

- A Whack-A-Mole game.

# Validation

- Please open `30 - Whack A Mole/index-START.html` on a browser.

- Click the button `START GAME!` to start playing.

- Click the moles as soon as they appear on their holes.

- The score board will be updated each time you successfully clicked a mole.

![image](https://user-images.githubusercontent.com/43773044/152471161-66511650-93b3-49d9-a541-48d3ea730e36.png)
